app/models/service_terraform_template.rb

```diff
@@ -75,8 +76,40 @@ def check_refreshed(_action)
 
   private
 
+  def job(action)
+    stack(action).miq_task.job
+  rescue err
```

Comment:
What exception are you expecting to catch here?  If it is just e.g. miq_task being `nil` then you're probably better off with `stack(action)&.miq_task&.job` without the exception handler+logging